### PR TITLE
Token request attribute configurable

### PR DESCRIPTION
**Pull request by basz**

### CODE REVIEW

**Comment by prolic** [comment]
On PHP 7.1: There was 1 failure:
1) ZfrOAuth2Test\Server\Grant\PasswordGrantTest::testCanCreateTokenResponse with data set #1 (false)
Failed asserting that 3599 matches expected 3600.

File: src/Middleware/ResourceServerMiddleware.php [hunk 1]
```diff
@@ -44,11 +45,17 @@ class ResourceServerMiddleware
...
     /**
+     * @var ServerOptions
+     */
+    private $serverOptions;
+
+    /**
      * @param ResourceServerInterface $resourceServer
      */
-    public function __construct(ResourceServerInterface $resourceServer)
+    public function __construct(ResourceServerInterface $resourceServer, ServerOptions $serverOptions)
```
Comment (prolic) [inline src/Middleware/ResourceServerMiddleware.php]: Maybe allow null for server options and use default then?

Comment (basz) [inline src/Middleware/ResourceServerMiddleware.php]: yeah, I saw too this but didn't want to change it all over the place again. 

But if I do I think it is be better to simply pass the option itself, unless there are more then 3 or 4 constructor arguments or the instance needs to pass them on to other dependencies. What do you think?

Comment (prolic) [inline src/Middleware/ResourceServerMiddleware.php]: Ok

**Pull request merged**